ui/app/controllers/allocations/allocation/index.js

```diff
@@ -47,10 +48,22 @@ export default class IndexController extends Controller.extend(Sortable) {
   }
 
   @computed('model.taskGroup.services.@each.name')
-  get services() {
+  get groupServices() {
     return (this.get('model.taskGroup.services') || []).sortBy('name');
   }
 
+  @computed('model.states.@each.services')
+  get taskServices() {
+    const allTaskServicesFragments =
+      this.get('model.states').mapBy('task.services').compact() || [];
+    const allTaskServices = allTaskServicesFragments
+      .map((frag) => frag.toArray())
+      .flat();
+    return allTaskServices.sortBy('name');
+  }
+
+  @union('taskServices', 'groupServices') services;
```

Comment:
`question`: We're going to use this entity to map over and visualize the status of our services, right? Currently, this won't re-compute the value of `services` because although we're marking these as computed properties, they're not updating in real-time.

We should fetch the data for `taskServices` and `groupServices` in the route and then mark those entities as `watchRelationship`. I don't think we'll be able to pick up registrations, updates, and de-registrations for this view right now. 

When I read the first paragraph of the Client State section of this [RFC](https://docs.google.com/document/d/1CBtnO1x6jAYcq_g3GUVEwhEDrMfV_i-kjPmxRpa5sm0/edit?usp=sharing), I think we're gonna need to consider being real-time from the very beginning of this project.